### PR TITLE
Add OCR example for supplementing

### LINKED ISSUE #1862: Be explicit that OCR should use supplementing motivation
OCR is a very common use case but isn't mentioned explicitly in the Presentation API spec (https://iiif.io/api/presentation/3.0/#35-values). We should amend descriptions of `supplementing` to add explicit mention of OCR. Editorial only

### PR DESCRIPTION
**Pull request by azaroth42**
Add OCR to 3.5 and 5.3 descriptions.

Closes #1862

### CODE REVIEW

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -1393,7 +1393,7 @@ Every Canvas _SHOULD_ have a `label` to display. If one is not provided, the cli
...
+Content that is derived from the Canvas, such as a manual or automatic (OCR) transcription of text in an image or the words spoken in an audio representation, _MUST_ be associated by an Annotation that has the `motivation` value `supplementing`. Annotations _MAY_ have any other `motivation` values as well. Thus content of any type may be associated with the Canvas via an Annotation that has the `motivation` value `painting`, meaning the content is part of the Canvas; an Annotation that has the `motivation` value `supplementing`, meaning the content is from the Canvas but not necessarily part of it; or an Annotation with another `motivation` meaning that it is somehow about the Canvas.
```
Comment (jpstroop) [inline source/api/presentation/3.0/index.md]: > Thus, 
(add comma)

Comment (jpstroop) [reply source/api/presentation/3.0/index.md]: Also, is it worth some italics here:
>meaning the content is part of the Canvas; an Annotation that has the `motivation` value `supplementing`, meaning the content is from the Canvas but not necessarily part of it

i.e.:
meaning the content is _part_ of the Canvas; an Annotation that has the `motivation` value `supplementing`, meaning the content is _from_ the Canvas but not necessarily part of it

**Comment by azaroth42** [comment]
Put "is" outside the italics.  Good to merge now?

**Comment by zimeon** [comment]
Just need @jpstroop to approve and merge

**Pull request merged**